test/metabase/actions/test_util.clj

```diff
@@ -236,17 +236,23 @@
     (something model-card-id id action-id model-id))
   nil)
 
-(defn do-with-actions-enabled
+(defn do-with-actions-set
```

Comment:
Would `do-with-enable-actions` be a better name? It's narrower and conveys the meaning of enabled/disabled.

This seems more natural to me:
`(do-with-enable-actions true ....)`
`(do-with-enable-actions false ....)`

Compared to this:
`(do-with-actions-set true ....)`
`(do-with-actions-set false ....)`

Since it is ambiguous what is being set by `do-with-actions-set`.

Reply:
I don't have big feelings about it, but I don't love the (English) grammar of `with enable actions`. I agree that `set` is a pretty generic term, but since it's only used in the context of `with-actions-enabled` or `with-actions-disabled` I think it's clear enough